runtime_config_linux.go

```diff
@@ -235,9 +235,14 @@ type Device struct {
 // Seccomp represents syscall restrictions
 type Seccomp struct {
 	DefaultAction Action     `json:"defaultAction"`
+	Architectures []Arch     `json:"architectures"`
 	Syscalls      []*Syscall `json:"syscalls"`
 }
 
+// Additional architectures permitted to be used for system calls
+// By default only the native architecture of the kernel is permitted
+type Arch string
```

Comment:
The only other one that i see is Operator.  Namespaces is a type because it has constants defined for it so its ok that it stays a type.  Maybe if we define operators for `type Operator string` it would be better.